deploy/crds/core.matrixorigin.io_logsets.yaml

```diff
@@ -6588,6 +6588,15 @@ spec:
                       type: object
                     type: array
                 type: object
+              pvcRetentionPolicy:
```

Comment:
Don't cn and dn need this parameter?

Reply:
Retention policies of CN/DN are `Delete` and are not tunable, since they are considered as cache.